skymp5-client/src/services/services/authService.ts

```diff
@@ -213,6 +216,53 @@ export class AuthService extends ClientListener {
       case events.updateRequired:
         this.sp.win32.loadUrl("https://skymp.net/UpdInstall");
         break;
+      case events.updateSkip:
+        browserState.comment = 'исправляем...';
+        this.sp.browser.executeJavaScript(new FunctionInfo(this.browsersideWidgetSetter).getText({ events, browserState, authData: authData }));
+
+        const client = new this.sp.HttpClient(settingsService.getMasterUrl());
```

Comment:
What do you think about `settingsService.makeMasterApiClient()`?

P.S. It also has a callback overload